src/ra_server.erl

```diff
@@ -1879,6 +1890,14 @@ handle_down(RaftState, snapshot_writer, Pid, Info,
 handle_down(RaftState, log, Pid, Info, #{log := Log0} = State) ->
     {Log, Effects} = ra_log:handle_event({down, Pid, Info}, Log0),
     {RaftState, State#{log => Log}, Effects};
+handle_down(RaftState, member_eval, _Pid, _Info, State) ->
+    #{member_eval_pid := _OldPid} = State,
+    %% Q: Unsure if this handling is needed, we could in ra_server_proc
+    %% just check if the process is alive?
+    %% Q: No effect is returned here. We should perhaps return an effect that
+    %% sets the timer to a lower value. But, that is currently done when the
+    %% spawned process sends its result
+    {RaftState, State#{member_eval_pid => undefined}, []};
```

Comment:
Set a short member_eval timeout here? Then we don't need any of the result handling.

Reply:
Done